pkg/controller/v1beta1/inferenceservice/controller_test.go

```diff
@@ -2088,3 +2091,15 @@ var _ = Describe("v1beta1 inference service controller", func() {
 		})
 	})
 })
+
+func TestGetEvent(t *testing.T) {
+
+	applyCmd := exec.Command("kubectl", "apply", "-f", "https://raw.githubusercontent.com/kserve/kserve/master/docs/samples/v1beta1/onnx/onnx.yaml")
```

Comment:
I think this is not the best practice to use a public url here as it might change. Can you find the best  practice to test event, maybe can refer [here](https://github.com/kubernetes/kubernetes/blob/fa66a37fd9bdf4e56bc86695dbdf0c7484ba86eb/staging/src/k8s.io/client-go/tools/record/event_test.go).

Reply:
Oops...I think I accidently commited while testing some ideas. Actually, we're trying to do it in non-url ways currently. Thanks for the tip!